rlberry/tests/test_agent.py

```diff
@@ -57,6 +58,7 @@ def test_finite_state_agent(agent):
     check_rlberry_agent(agent, env="discrete_state")
 
 
+@pytest.mark.xfail(sys.platform == "win32", reason="bug with windows???")
```

Comment:
Maybe we can do another test for windows specifically, in which we don't test PPO. With this xfail, you are xfailing the test for all agents even though only one agent fails (I think)